boards/m5stack/m5stack_cores3/m5stack_cores3-pinctrl.dtsi

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 Zhang Xingtao <zhxt@live.cn>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/esp-pinctrl-common.h>
+#include <dt-bindings/pinctrl/esp32s3-pinctrl.h>
```

Comment:
This is a bit weird the `include` path.
 
https://github.com/zephyrproject-rtos/zephyr/pull/79765/commits/931c48490f2a66e5e966bb2d7d196bbdd455a14c

Reply:
> This is a bit weird the `include` path.
> 
> [931c484](https://github.com/zephyrproject-rtos/zephyr/commit/931c48490f2a66e5e966bb2d7d196bbdd455a14c)

Yeah, fixed.